utils/tempered.c

```diff
@@ -30,6 +31,9 @@ void show_help()
 "Known options:\n"
 "    -h\n"
 "    --help                 Show this help text\n"
+"    -t\n"
+"    --type                 Returns only the numeric value of the sensor.\n"
```

Comment:
The required argument should be mentioned here in angle brackets.
`--type`
->
`--type <type>`